src/main/java/mate/academy/hibernate/relations/model/Actor.java

```diff
@@ -1,8 +1,24 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.JoinTable;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "actors")
 public class Actor implements Cloneable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String name;
+    @ManyToOne
+    @JoinTable(name = "actors_country", joinColumns = @JoinColumn(name = "actors_id"),
+            inverseJoinColumns = @JoinColumn(name = "country_id"))
```

Comment:
there is no need for the third table when we have `many-to-one` or `one-to-many` relationship, in that case the entity that is on `many` side of it - will just have a column in it's table called for example `country_id`.
Having a third table may be ok too, but usually if there is no needing that - better to avoid it